pkg/genericapiserver/storage_factory.go

```diff
@@ -268,3 +285,30 @@ func NewStorageCodec(storageMediaType string, ns runtime.StorageSerializer, stor
 	}
 	return runtime.NewCodec(encoder, decoder), nil
 }
+
+var specialDefaultResourcePrefixes = map[unversioned.GroupResource]string{
```

Comment:
> How do we prove we don't regress on old clusters here? Will 1.3 -> 1.4 deployment catch us.

I had to find these manually based on code inspection.  Going forward, everyone will be using their resources, to-lowered.  I would suggest that for an etcd3 migration, we consider changing directories to subdivide by API group.